source/core/options.ts

```diff
@@ -1763,12 +1763,12 @@ export default class Options {
 
 	@default true
 	*/
-	get followRedirect(): boolean {
+	get followRedirect(): boolean | ((response: PlainResponse) => boolean) {
```

Comment:
The function needs to be documented (with text) here and in the readme.

Reply:
By documented you mean I should add a note how to use it?